src/components/register/steps/personal.tsx

```diff
@@ -16,9 +17,15 @@ const languageOptions = [...Object.entries(langMap)]
 
 
 const Personal = (_: RouteComponentProps) => {
-	const { register, handleSubmit } = useForm<PersonalInfo>()
+	const { register, watch, handleSubmit } = useForm<PersonalInfo>()
 	const dispatch = useAppDispatch()
 
+	useEffect(() => {
+		const subscription = watch(data => dispatch(ChangePersonalInfo.create(data)))
+
+		return () => subscription.unsubscribe()
```

Comment:
Since we need to return a function, can we not just return `subscription.unsubscribe`?